### ISSUE TITLE
Template in openshift namespace show up twice when Template Service Broker installed

### ISSUE THREAD

**Issue by fortinj66** [openshift/origin#19462]
Template show up twice when Template Service Broker installed

##### Version
oc version
oc v3.7.1+c2ce2c0-1
kubernetes v1.7.6+a08f5eeb62
features: Basic-Auth GSSAPI Kerberos SPNEGO

##### Steps To Reproduce
Install Origin 3.7 with Service Catalog and default templates installed

##### Current Result
Templates installed in openshift namespace show up twice in UI

##### Expected Result
Templates should only show up once.

##### Additional Information
If I uninstall the Template Service Broker the templaces only show up once...

**Comment by fortinj66** [comment]
Hi Jessica, 

The cluster was installed/upgraded with openshift-ansible and the Service Catalog, ASB, and TSB installed with ansible after the cluster was installed.  There were several issues with ASB which led to the Service Catalog and other components being removed and reinstalled several times.

I have also created a new 3.7 cluster using openshift-ansible with the same result.

What portion of the console config controls the pre/post TSB configuration?

**Comment by spadgett** [comment]
Please make sure the following file exists:

```
/etc/origin/master/openshift-ansible-catalog-console.js
```

With this content:

```js
window.OPENSHIFT_CONSTANTS.TEMPLATE_SERVICE_BROKER_ENABLED = 'true';
```

It must also be set as an extension in master-config.yaml:

```yaml
assetConfig:
  extensionScripts:
    - '/etc/origin/master/openshift-ansible-catalog-console.js'
```

I believe there is an installer bug where the extension doesn't always get set up properly, although that's been fixed indirectly in 3.9 since the extension is no longer needed.

**Comment by fortinj66** [comment]
@spadgett 

Thanks!!  That did the trick!

--John
